libraries/AP_Math/control.cpp

```diff
@@ -385,13 +306,12 @@ void shape_pos_vel_accel_z(const Vector3f& pos_input, const Vector3f& vel_input,
  The time constant defines the acceleration error decay in the kinematic path as the system approaches constant acceleration.
  The time constant also defines the time taken to achieve the maximum acceleration.
  The time constant must be positive.
- The function alters the input position to be the closest position that the system could reach zero acceleration in the minimum time.
- This function operates on the x and y axis of both Vector2f or Vector3f inputs.
+ The function alters the accel to be jerk limited
```

Comment:
The function alters the variable accel to follow a jerk limited kinematic path to the variables pos_input, vel_input and accel_input

Reply:
fixed